pandas/core/indexes/multi.py

```diff
@@ -281,6 +294,9 @@ def _verify_integrity(self, codes=None, levels=None):
                                  " level  (%d). NOTE: this index is in an"
                                  " inconsistent state" % (i, level_codes.max(),
                                                           len(level)))
+            if len(level_codes) and level_codes.min() < -1:
```

Comment:
pls use new style formatting, e.g. .format()

Reply:
you can change the other usages in this function as well

Reply:
Thanks, I have reformatted the error string